homeassistant/components/sensor/forecast.py

```diff
@@ -95,20 +88,10 @@ def __init__(self, weather_data, sensor_type):
         """Initialize the sensor."""
         self.client_name = 'Weather'
         self._name = SENSOR_TYPES[sensor_type][0]
-        self.forecast_client = weather_data
+        self.sdk = weather_data
```

Comment:
No "real" reason for this change. I was attempting to do a larger refactor at one point and this made it in. 

You can see that the encapsulated logic was being called `forecast_client`, `weather_data`, `data`, and `ForeCastData` depending on which block of code you were looking in. Looking at it, I think it might actually be better to just consistently use the terms `ForeCastData` and `forecast_data` everywhere. (or `...client/Client`) Trying to make it more obvious that a call to specific method is potentially going to result in an API call. Thus hopefully avoiding future dev error.

let me know if you have a preference.